src/Bicep.Local.Extension.UnitTests/Bicep.Local.Extension.UnitTests.csproj

```
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

We default most of these settings at the solution level

```suggestion
    <ImplicitUsings>enable</ImplicitUsings>
```